wee_slack.py

```diff
@@ -3311,6 +3314,40 @@ def linkify_word(match):
     return re.sub(linkify_regex, linkify_word, message_escaped, re.UNICODE)
 
 
+def unfurl_blocks(message_json):
+    block_text = []
+    for block in message_json["blocks"]:
+        try:
+            if block["type"] == "section":
+                if "text" in block: block_text += unfurl_texts([block["text"]])
+                if "fields" in block: block_text += unfurl_texts(block["fields"])
+            elif block["type"] == "actions":
+                block_text.append("|".join(i["text"]["text"] for i in block["elements"]))
+            elif block["type"] == "call":
+                block_text.append(". Join via ").append(block["call"]["v1"]["join_url"])
```

Comment:
Calling `.append` twice like this doesn't work since `.append` returns `None`. @gianarb has fixed this in https://github.com/wee-slack/wee-slack/pull/742/commits/4ec9aa586c480f8606795b9f261d84a8bb6a2497.

That also drops the `rich_text` type which we want to do, so we don't print `Unsupported block type` after each message sent from the web client. I saw you said you don't feel quite right just dropping the content, but Slack has said the `text` field will still contain the message[0], and we don't have anything to do with the field unless we use it to render the message (and then we would have to drop the text field, since we don't want the message twice).

Can you pull in that commit to this PR (except for the last part which changes the else-block)? Or can/should I pull it into this PR?

[0]: From https://api.slack.com/changelog/2019-09-what-they-see-is-what-you-get-and-more-and-less:
> The text field will continue to contain a textual and markup-laden representation of the user's typed message.
> Messages are still messages. These ones are still written by users. You'll still find the essence of their communication in the text field of message objects.